MbDotNet/Models/Responses/ProxyResponse.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+using MbDotNet.Models.Responses.Fields;
+
+namespace MbDotNet.Models.Responses
+{
+    public class ProxyResponse<T> : ResponseBase where T : ResponseFields, new()
```

Comment:
Please add a test for this. It's not super helpful, but it will at least maintain consistency with the other response classes. See `IsResponseTests`.